docs/getting-started.md

```diff
@@ -60,22 +60,22 @@ export VDC_REGION=us
 ```
 
 ### Start the backend and frontend
-1. Install dependencies with
+1. Install dependencies with:
 ``` bash
 yarn install
 ```
 
-2. Build the frontend application
+2. Build the frontend application:
 ``` bash
 yarn run build
```

Comment:
I think with yarn, it's just `yarn build && yarn start` right?

Reply:
With angular it's weird because it's two different servers. Just tested this and I don't think we wouldn't even need to run build. Start does the build for us.